lib/transformer.js

```diff
@@ -177,8 +177,8 @@ class Transformer {
     for (const obj of interfaces.values()) {
       let source = obj.toString();
 
-      const implDir = path.relative(outputDir, obj.opts.implDir).replace(/\\/g, "/"); // fix windows file paths
-      let implFile = implDir + "/" + obj.name + this.ctx.implSuffix;
+      const implDir = obj.opts.implDir.replace(/\\/g, "/"); // fix windows file paths
+      let implFile = path.relative(outputDir, path.resolve(implDir, obj.name + this.ctx.implSuffix));
```

Comment:
If you're calling relative/resolve here, I think you're reintroducing windows slashes you'll need to correct.

Reply:
Ah I see. Moved Windows path fix to last.